docs/cloud/dev/spryker-cloud-commerce-os/add-variables-in-the-parameter-store.md

```diff
@@ -20,6 +20,19 @@ This feature is part of a gradual rollout and will be available to everyone even
 
 {% endinfo_block %}
 
+## Customer-Owned and Spryker-Owned Variables
+
+When working with environment variables in the Spryker Cloud Commerce OS, it's important to understand that there are two types of variables: those owned by the customer and those owned by Spryker.
+
+Customer-owned variables are created and managed by you - the onboarded customer or implementation partner. You have full control over these variables and can add or edit them according to your needs. However, please note that changes to these variables do not automatically propagate into the running environment. To apply changes made to your environment variables, you'll need to run an ECS-updater-* pipeline in order to bring them to the containers (full re-deploy would also do the trick, of course). This process does not require Spryker Cloud or support intervention.
```

Comment:
@feversocial "This process does not require Spryker Cloud or support intervention." - what does it mean "does not require Spryker Cloud intervention"?